src/test/ui/async-await/unresolved_type_param.stderr

```diff
@@ -34,6 +34,30 @@ note: the type is part of the `async fn` body because of this `await`
 LL |     bar().await;
    |     ^^^^^^^^^^^
 
-error: aborting due to 3 previous errors
+error[E0698]: type inside `async fn` body must be known in this context
```

Comment:
Can you file a follow-up issue to make this error message better by removing the duplicates?

Reply:
Filed https://github.com/rust-lang/rust/issues/91463.

At various points trying to fix the generator captures issue, I think was able to remove some of the duplicates, so maybe that's a good place to look for a solution.